pyproject.toml

```diff
@@ -112,19 +93,17 @@ exclude_lines = [
   "if TYPE_CHECKING",
 ]
 
-[tool.black]
-target-version = ['py37']
-line-length = 100
-
 # https://github.com/charliermarsh/ruff
 [tool.ruff]
-target-version = "py37"
-line-length = 100
-show-source = true
-src = ["ConfigSpace", "test"]
+target-version = "py38"
+line-length = 88
+output-format = "full"
+src = ["src", "test"]
```

Comment:
Most packaging tutorials now recommend a `src` distribution. Also prevents issues with incorrect setups where if you run `pytest` within the root directory and theres a test with `import ConfigSpace`, it's actually running the installed version rather than doing a namespaced lookup of tests.